docs/runbooks/setup-vertex-tryon.md

```
@@ -0,0 +1,129 @@
+Keep the key out of the repo. Default location: `~/.config/gcloud/ptso-tryon.json`.
+
+```bash
+mkdir -p ~/.config/gcloud
+gcloud iam service-accounts keys create ~/.config/gcloud/ptso-tryon.json \
+    --iam-account=ptso-tryon@ptso-agent.iam.gserviceaccount.com
+chmod 600 ~/.config/gcloud/ptso-tryon.json
+```
+
+Set in your local `.env`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)